Dockerfile

```diff
@@ -11,8 +11,11 @@ RUN apt-get update && \
 
 # BESS pre-reqs
 WORKDIR /bess
-ARG BESS_COMMIT=dpdk-2011-focal
-RUN git clone https://github.com/omec-project/bess.git .
+#ARG BESS_COMMIT=dpdk-2011-focal
+#RUN git clone https://github.com/omec-project/bess.git .
+# Temporary: To test CNDP BESS with OMEC-UPF.
+ARG BESS_COMMIT=feat_cndp_bess_port
+RUN git clone https://github.com/manojgop/bess.git .
```

Comment:
You can revert this change, I just merged the BESS PR related to CNDP

Reply:
shouldn't it be `https://github.com/omec-project/bess.git` instead of `https://github.com/NetSys/bess.git` in  https://github.com/omec-project/upf/blob/master/.github/workflows/manual.yml#L25

Reply:
Done. But looks like we need to generate updated BESS docker image for the workflow manual.yml to run CI test for this PR

Reply:
Yes, let me take a look at the entire repo to see where else this needs to be updated.

Reply:
Done. I reviewed the repo to make sure there were no more links pointing to NetSys and a new image was created